Formula/paps.rb

```diff
@@ -53,7 +53,7 @@ def install
 --- a/src/libpaps.c
 +++ b/src/libpaps.c
 @@ -25,8 +25,10 @@
- 
+
```

Comment:
Was this caused by a rubocop autofix? If so, that is a problem with the autofix not the formula.